dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -664,18 +664,23 @@ class StartupTest {
           ]);
           applicationBinaryPath = '$testDirectory/build/app/outputs/flutter-apk/app-profile.apk';
           break;
+        case DeviceOperatingSystem.fake:
+        case DeviceOperatingSystem.fuchsia:
+          break;
         case DeviceOperatingSystem.ios:
+        case DeviceOperatingSystem.macos:
+          final String deviceName = <DeviceOperatingSystem, String>{
+            DeviceOperatingSystem.macos: 'macos',
+            DeviceOperatingSystem.ios: 'ios',
+          }[deviceOperatingSystem]!;
           await flutter('build', options: <String>[
-            'ios',
+            deviceName,
```

Comment:
How about instead of the map above you check with a ternary:
```suggestion
            DeviceOperatingSystem.ios ? 'ios' : 'macos',
```